mender-deb-package

```diff
@@ -134,12 +134,13 @@ prepare_recipe() {
     if [ "${debian_recipe}" = "debian-2.1.x" ]; then
       # Skip for 2.1.x, special handling in debian/rules
       true
-    elif [ "${debian_recipe}" = "debian-master" ]; then
-      # skip
-      true
-    else
+    elif [[ "${debian_recipe}" =~ ^debian-2.*  || "${debian_recipe}" =~ ^debian-3.* ]]; then
```

Comment:
Q: Shouldn't the `wildcard` be quoted?

Reply:
You can also do: ` =~ ^debian-(2|3).*` to make it shorter :)

Reply:
At least on my version of bash, when quoting it does string comparison and not regex. There are also other non-quoted regex here in this file.

I will modify it with your fancy suggestion but not quote it :+1:

Reply:
Yup, I guess it makes sense. Bash is `magic` that is for sure :magic: